gcp_variant_transforms/libs/bigquery_vcf_schema.py

```diff
@@ -82,8 +83,15 @@ class _TableFieldConstants(object):
 _JSON_CONCATENATION_OVERHEAD_BYTES = 5
 
 
+# TODO(bashir2): Using type identifiers like ``HeaderFields`` does not seem
```

Comment:
yeah, the type identifiers have been on my mind for some time. I inherited it from Beam conventions, but have not actually set up automated documentation generation to verify that it finds the correct class.
I think the 'correct' way is something like `` :class:`HeaderFields` ``. In any case, I filed Issue #108 for this. You can remove the TODO from here as it's awkward to single out this particular field.